src/lib/tuning/index/column_ref.hpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "types.hpp"
+
+namespace opossum {
+
+/**
+ * A ColumnRef is used to reference a set of columns by a table name
+ * and their column indexes in this table.
+ * This is used by the IndexEvaluator, IndexChoices and IndexOperations.
+ */
+struct ColumnRef {
```

Comment:
The name is unfortunate, since we already have the "LQPColumnReference". This struct looks more like a "Named(Table?)ColumnSet" to me. Also, you really shouldn't have to overload all these operators, < and == should be sufficient

Reply:
renamed to `IndexableColumnSet`, removed operator overloading for everything but `<` and `==`